wurst/systems/trade/TradeShipSelection.wurst

```diff
@@ -0,0 +1,18 @@
+package TradeShipSelection
+import RegisterEvents
+import ClosureForGroups
+import ClosureTimers
+
+function onSelect()
+    let selected = GetTriggerUnit()
+    let trigPlayer = GetTriggerPlayer()
+
+    if not IsUnitTradeShip(selected)
```

Comment:
How is `selected` single unit? What happens if you have two units selected (your hero and ally A's) and you select two more (ally B's and a ship)?